src/player.js

```diff
@@ -482,6 +484,35 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * Checks for callbacks that should change the tracks, and call them on the
+   * respective track group (audio/text/video)
+   * @private
+   * @returns {void}
+   */
+  _maybeSetTracksLabels() {
+    const labelsCallbacks = this._config.customLabels;
```

Comment:
call it simply `customLabels`
`const customLabels = this._config.customLabels;`

Reply:
agree, changed.